src/summary/LongitudinalTableVisualizer.jsx

```diff
@@ -43,7 +64,7 @@ export default class LongitudinalTableVisualizer extends Visualizer {
     render() {
         return (
             <div>
-                <LongitudinalTable dataInfo={this.state.data} tdpSearchSuggestions={this.props.tdpSearchSuggestions} conditionSectionName={this.props.conditionSection.name} subsectionLabel={this.props.conditionSection.subsectionLabel}/>
+                <LongitudinalTable onClick={this.onClick} dataInfo={this.state.data} tdpSearchSuggestions={this.props.tdpSearchSuggestions} conditionSectionName={this.props.conditionSection.name} subsectionLabel={this.props.conditionSection.subsectionLabel} />
```

Comment:
[JA2]: This is pretty small as well, so we can talk about if it's something we want to change, but since this function gets passed onto LongitudinalTable and called from there, we might want to consider renaming it to something other than `onClick` to reflect what it's actually doing. Since it's not being used as an event handler here, we don't need to call it onClick, and you should be able to rename the prop and the function itself.